cli/state.go

```diff
@@ -50,7 +51,7 @@ func statePull() *cobra.Command {
 			}
 
 			if len(args) < 2 {
-				cmd.Println(string(state))
+				fmt.Fprintln(cmd.OutOrStdout(), string(state))
```

Comment:
Linting can be fixed with:
```suggestion
				_, _ = fmt.Fprintln(cmd.OutOrStdout(), string(state))
```

Reply:
tyty